pkg/services/dashboards/models.go

```diff
@@ -302,9 +302,10 @@ type DeleteOrphanedProvisionedDashboardsCommand struct {
 //
 // Multiple constraints can be combined.
 type GetDashboardQuery struct {
-	ID       int64
-	UID      string
-	Title    *string
+	ID    int64
+	UID   string
+	Title *string
+	// Deprecated: use FolderUID instead
```

Comment:
Nit: might be helpful to change the phrasing to use folder UID instead. To me FolderUID sounds like the name of a field but FolderUID doesn't exist as a field. What do you think?

```suggestion
	// Deprecated: use folder UID instead
```

Reply:
ahh i see your point! i think i'll keep it as FolderUID for now since the next step will be adding in FolderUID fields (also for consistency). but for a future case i'll def go with what you suggested!

Reply:
That makes total sense 👍